class.jetpack.php

```diff
@@ -7364,14 +7373,13 @@ public static function load_jetpack_gutenberg() {
 			$version
 		);
 
-		wp_register_style(
-			'jetpack-blocks-editor',
-			$editor_style,
-			array(),
-			$version
-		);
+		wp_register_style( 'jetpack-blocks-editor', $editor_style, array(), $version );
+		wp_register_script( 'jetpack-blocks-view', $view_script, array(), $version );
```

Comment:
We eventually might want to audit the dependencies here, as some of the blocks might end up needing specific WordPress packages or another script.